src/Microsoft.Sbom.Api/Config/ConfigSanitizer.cs

```diff
@@ -25,6 +26,10 @@ public class ConfigSanitizer
         private readonly IFileSystemUtils fileSystemUtils;
         private readonly IAssemblyConfig assemblyConfig;
 
+        internal static string SBOMToolVersion => VersionValue.Value;
+
+        private static readonly Lazy<string> VersionValue = new Lazy<string>(() => typeof(SbomToolCmdRunner).GetTypeInfo().Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion ?? string.Empty);
```

Comment:
Is this how we are retrieving the version in other places?

Reply:
Yes this the same as in the --version parameter. Would it be better to store it in the Constants?